### PR TITLE
IMSC 1.1 CR2

### LINKED ISSUE #414: Update SOTD for CR2
As per https://w3c.github.io/spec-releases/milestones/?cr=2018-07-26 For the planned publication date of July 26th the deadline for comments on the CR needs to be set at 23rd August and the earliest transition to PR is 6th September.

### PR DESCRIPTION
**Pull request by palemieux**
Close #414

### CODE REVIEW

**Comment by palemieux** [comment]
Built at https://rawgit.com/w3c/imsc/imsc1.1-cr2/imsc1/spec/ttml-ww-profiles.html

File: imsc1/spec/substantive-changes-summary.txt [hunk 1]
```diff
@@ -12,6 +12,32 @@ http://services.w3.org/htmldiff across published versions of the specification.
 NOTE 2: Issues and their dispositions are documented at https://github.com/w3c/imsc/issues
 
 
+From CR (20180503)
+==================
+
+* Make reference to TTML2 dated (#381)
+
+* Apply tt:initial to extension style properties (#386)
```
Comment (nigelmegitt) [inline imsc1/spec/substantive-changes-summary.txt]: Suggest "Permit extension style properties to be used in tt:initial"?

File: imsc1/spec/substantive-changes-summary.txt [hunk 2]
```diff
@@ -12,6 +12,32 @@ http://services.w3.org/htmldiff across published versions of the specification.
...
+
+* Add support for #length-root-container-relative (rh/rw units) (#372)
+
+* Add support for tts:rubyAlign='spaceAround' (#402)
```
Comment (nigelmegitt) [inline imsc1/spec/substantive-changes-summary.txt]: Shouldn't the quotes be double rather than single?

**Comment by tmichel07** [comment]
uploaded at 
      
    https://www.w3.org/TR/2018/CR-ttml-imsc1.1-20180726/
    Thierry
    
    
    On 05/07/2018 18:17, Pierre-Anthony
      Lemieux wrote:
    
    
      Built at https://rawgit.com/w3c/imsc/imsc1.1-cr2/imsc1/spec/ttml-ww-profiles.html
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view it on GitHub, or mute the thread.
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/w3c/imsc","title":"w3c/imsc","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/w3c/imsc"}},"updates":{"snippets":[{"icon":"PERSON","message":"@palemieux in #415: Built at https://rawgit.com/w3c/imsc/imsc1.1-cr2/imsc1/spec/ttml-ww-profiles.html"}],"action":{"name":"View Pull Request","url":"https://github.com/w3c/imsc/pull/415#issuecomment-402776350"}}}
      [
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/w3c/imsc/pull/415#issuecomment-402776350",
"url": "https://github.com/w3c/imsc/pull/415#issuecomment-402776350",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [w3c/imsc] IMSC 1.1 CR2 (#415)",
"sections": [
{
"text": "",
"activityTitle": "**Pierre-Anthony Lemieux**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@palemieux",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"w3c/imsc\",\n\"issueId\": 415,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close pull request",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"PullRequestClose\",\n\"repositoryFullName\": \"w3c/imsc\",\n\"pullRequestId\": 415\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/w3c/imsc/pull/415#issuecomment-402776350"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 351611499\n}"
}
],
"themeColor": "26292E"
}
]

**Pull request merged**